package DB연결;

public class 에러처리 {

	public static void main(String[] args) throws Exception {
		// TODO Auto-generated method stub
		//try {
			int x = 10/0;			//에러날수있는코드
		//} catch (Exception e) {
			// TODO: handle exception
			//System.out.println("산술연산부분에서 에러남"); //에러시대체방법
		//}
		
		System.out.println("내가 실행될까요");
	}

}
